### DISCUSSION TITLE
Nuclei doesn't find anything

### DISCUSSION THREAD

**Discussion by Ss4x** [projectdiscovery/nuclei#2692] [Q&A]
I tested the nuclei at http://testphp.vulnweb.com/ and it did not return any vulnerabilities

![image](https://user-images.githubusercontent.com/115406109/194757291-b9587707-cba9-4acc-b03a-d7684459e8f8.png)

**Comment by olearycrew** [comment]
This is expected behavior - the sub-pages of testphp web have detectable vulnerabilities.
